x-pack/solutions/search/plugins/search_indices/public/code_examples/python.ts

```diff
@@ -133,8 +133,39 @@ mapping_response = client.indices.put_mapping(index=index_name, body=mappings)
 print(mapping_response)
 `;
 
+const semanticIngestCommand: IngestCodeSnippetFunction = ({
+  elasticsearchURL,
+  apiKey,
+  indexName,
+  sampleDocuments,
+}) => `from elasticsearch import Elasticsearch, helpers
+
+# Set the ingestion timeout to 5 minutes, to allow time for semantic ingestion
+# to complete. The initial ingest with semantic_text fields may take longer
+# while the model loads, or ML nodes are allocated.
+ingestion_timeout=300
+
+client = Elasticsearch(
+    "${elasticsearchURL}",
+    api_key="${apiKey ?? API_KEY_PLACEHOLDER}",
+    request_timeout=ingestion_timeout,
```

Comment:
Now looking at the notebooks I _think_ maybe I should update to set this with `client.options()` when calling `helpers.bulk()`

Reply:
If this client is going to be used for other things besides this potentially slow `bulk` call, then yes, maybe you should initialize the client with the default timeout (which I believe is 10s), and then pass `client.options(request_timeout=300)` on the call to `bulk()`.